pkg/datarepair/checker/checker.go

```diff
@@ -251,30 +251,32 @@ func (checker *Checker) updateSegmentStatus(ctx context.Context, pointer *pb.Poi
 	return nil
 }
 
-// IrreparableProcess picks items from irreparabledb and add them to the repair
-// worker queue if they, now, can be repaired.
+// IrreparableProcess iterates over all items in the irreparabledb. If an item can
+// now be repaired then it is added to a worker queue.
 func (checker *Checker) IrreparableProcess(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
-
-	limit := 1
-	var offset int64
+	const limit = 1000
```

Comment:
why 1000 ?

Reply:
i just picked a number that isn't too big or too small. i was thinking either 1000 or 10,000. you have opinions?

Reply:
should we make it configurable ? I typically find magic numbers to be a bad pattern.

Reply:
i dont think it needs to be configurable. i think we should pick what is most performant.

Reply:
agreed but how are we going to determine most performant ?

Reply:
im trying to test that right now. I have a dump of the irreparable table with 200k rows that im testing queries on. i will post the details once i have some findings.

Reply:
👍 I assumed it would be easier to determine the most performant choice if it was a configurable number

Reply:
im just running queries against the db table directly so its easy

Reply:
i did some testing by running queries with `explain analyze` against the irreparable table with 230k rows (a dump from stefan's prod SA). This showed that that a limit of 1k is four times faster than 10k. The query plan with 1k limit does an index scan, while the query plan of 10k changes to do a bitmap index/heap scan. While these results are specific to this table and the size, I think its fine to hard code 1k limit for now.  If the performance is still a bottleneck in the future we can always modify downt the road.